networkx/algorithms/components/strongly_connected.py

```diff
@@ -371,6 +371,26 @@ def condensation(G, scc=None):
     NetworkXNotImplemented
         If G is undirected.
 
+    Examples
+    -----
+    >>> G = nx.barbell_graph(4, 0)
+    >>> G.remove_edge(3, 4)
+    >>> G = nx.DiGraph(G)
+    >>> H = nx.condensation(G)
+    >>> H.nodes.data()
+    NodeDataView({0: {'members': {0, 1, 2, 3}}, 1: {'members': {4, 5, 6, 7}}})
+    >>> H.graph['mapping']
+    {0: 0, 1: 0, 2: 0, 3: 0, 4: 1, 5: 1, 6: 1, 7: 1}
+
+    >>> G = nx.complete_graph(7)
+    >>> G = nx.DiGraph(G)
```

Comment:
There is no need to cast the `DiGraph`, for complete graphs we can directly get a `DiGraph` with `create_using`.
`G = nx.complete_graph(7, create_using=nx.DiGraph)` should do the job :)

Reply:
Oh nice, i haven't used this function ever. Thanks for the clarification.